### ISSUE TITLE
Karma seems to fail on Node v15 (util.inspect issue)

### ISSUE THREAD

**Issue by jasonwilliams** [karma-runner/karma#3587]
I'm not sure why i hit this issue, but it seems to fail here.
Here's a stack trace.

```
TypeError: Function has non-object prototype 'null' in instanceof check
    at Function.[Symbol.hasInstance] (<anonymous>)
    at getConstructorName (node:internal/util/inspect:545:13)
    at formatRaw (node:internal/util/inspect:813:23)
    at formatValue (node:internal/util/inspect:803:10)
    at formatProperty (node:internal/util/inspect:1689:11)
    at formatRaw (node:internal/util/inspect:1017:9)
    at formatValue (node:internal/util/inspect:803:10)
    at formatProperty (node:internal/util/inspect:1689:11)
    at formatRaw (node:internal/util/inspect:1017:9)
    at formatValue (node:internal/util/inspect:803:10)
    at formatProperty (node:internal/util/inspect:1689:11)
    at formatArray (node:internal/util/inspect:1519:17)
    at formatRaw (node:internal/util/inspect:1014:14)
    at formatValue (node:internal/util/inspect:803:10)
    at formatProperty (node:internal/util/inspect:1689:11)
    at formatRaw (node:internal/util/inspect:1017:9)
    at formatValue (node:internal/util/inspect:803:10)
    at formatProperty (node:internal/util/inspect:1689:11)
    at formatRaw (node:internal/util/inspect:1017:9)
    at formatValue (node:internal/util/inspect:803:10)
    at Object.inspect (node:internal/util/inspect:336:10)
    at new Server (/app/node_modules/karma/lib/server.js:67:41)
    at Object.exports.run (/app/node_modules/karma/lib/cli.js:258:7)
    at Object.<anonymous> (/app/node_modules/karma/bin/karma:3:23)
    at Module._compile (node:internal/modules/cjs/loader:1083:30)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1112:10)
    at Module.load (node:internal/modules/cjs/loader:948:32)
    at Function.Module._load (node:internal/modules/cjs/loader:789:14)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:72:12)
    at node:internal/main/run_main_module:17:47
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

It seems to be this line which triggers the error above.
https://github.com/karma-runner/karma/blob/master/lib/server.js#L66

Looks related to https://github.com/nodejs/node/issues/35730 ? But I'm not sure.

Karma v 5.2.3
Node v15.0.1

**Comment by devoto13** [comment]
Well, it indeed looks like a Node bug, which should be (and was) fixed in Node. So we just need to wait for the next Node release. Or what what is your expectation here?

**Comment by johnjbarton** [comment]
We test on v14 LTS so the best fix is to use v14.

**Comment by devoto13** [comment]
This has been fixed in Node 15.5 (see [changelog](https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V15.md#2020-12-22-version-1550-current-targos)).
